internal/server/handlers.go

```diff
@@ -145,7 +145,7 @@ func (a *API) Signup(c *gin.Context, r *api.SignupRequest) (*api.SignupResponse,
 	a.t.Event("signup", identity.ID.String(), Properties{})
 
 	err = email.SendSignupEmail("", r.Name, email.SignupData{
-		Link: fmt.Sprintf("https://%s/login", r.Org.Subdomain),
+		Link: fmt.Sprintf("https://%s/login", suDetails.Org.Domain),
```

Comment:
If I'm reading this correctly, `r.Org.Subdomain` only contains the subdomain part but the details contains a FQDN?

Reply:
yes

Reply:
Can be in a follow up PR but should add a test case for this